docs/UserGuide.md

```diff
@@ -44,8 +44,8 @@ With features such as the ability to import your friends' schedules, find friend
 
 3. Copy the file to the folder you want to use as the _home folder_ for your UniGenda.
 
-4. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
-   ![Ui](images/Ui.png)
+4. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data. Also note the name for each section of the UI, as it will be referenced throughout the User Guide.<br>
```

Comment:
```suggestion
4. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Also note the name for each section of the UI, as it will be referenced throughout the User Guide.<br>
```
I don't think we need this part. This is already explained in FAQ